.github/workflows/ci.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           PGHOST: localhost
 
       - name: Publish coverage report
-        uses: codecov/codecov-action@v1
+        uses: codecov/codecov-action@v4
```

Comment:
this doesn't work out of the box since v4 now requires you to pass the token explicitly